src/SharedFrameworkValidation/ApiCompatBaseline.LKGRuntime.CurrentRuntime.txt

```diff
@@ -0,0 +1,875 @@
+TypesMustExist : Type 'Microsoft.CodeAnalysis.CSharp.Syntax.ImplicitElementAccessSyntax' does not exist in the implementation but it does exist in the contract.
+TypesMustExist : Type 'Microsoft.CodeAnalysis.CSharp.Syntax.IncompleteMemberSyntax' does not exist in the implementation but it does exist in the contract.
+TypesMustExist : Type 'Microsoft.CodeAnalysis.CSharp.Syntax.IndexerDeclarationSyntax' does not exist in the implementation but it does exist in the contract.
+TypesMustExist : Type 'Microsoft.CodeAnalysis.CSharp.Syntax.IndexerMemberCrefSyntax' does not exist in the implementation but it does exist in the contract.
+TypesMustExist : Type 'Microsoft.CodeAnalysis.CSharp.Syntax.InitializerExpressionSyntax' does not exist in the implementation but it does exist in the contract.
+TypesMustExist : Type 'Microsoft.CodeAnalysis.CSharp.Syntax.InstanceExpressionSyntax' does not exist in the implementation but it does exist in the contract.
+TypesMustExist : Type 'Microsoft.CodeAnalysis.CSharp.Syntax.InterfaceDeclarationSyntax' does not exist in the implementation but it does exist in the contract.
+TypesMustExist : Type 'Microsoft.CodeAnalysis.CSharp.Syntax.InterpolatedStringContentSyntax' does not exist in the implementation but it does exist in the contract.
+TypesMustExist : Type 'Microsoft.CodeAnalysis.CSharp.Syntax.InterpolatedStringExpressionSyntax' does not exist in the implementation but it does exist in the contract.
+TypesMustExist : Type 'Microsoft.CodeAnalysis.CSharp.Syntax.InterpolatedStringTextSyntax' does not exist in the implementation but it does exist in the contract.
+TypesMustExist : Type 'Microsoft.CodeAnalysis.CSharp.Syntax.InterpolationAlignmentClauseSyntax' does not exist in the implementation but it does exist in the contract.
+TypesMustExist : Type 'Microsoft.CodeAnalysis.CSharp.Syntax.InterpolationFormatClauseSyntax' does not exist in the implementation but it does exist in the contract.
+TypesMustExist : Type 'Microsoft.CodeAnalysis.CSharp.Syntax.InterpolationSyntax' does not exist in the implementation but it does exist in the contract.
+TypesMustExist : Type 'Microsoft.CodeAnalysis.CSharp.Syntax.InvocationExpressionSyntax' does not exist in the implementation but it does exist in the contract.
+TypesMustExist : Type 'Microsoft.CodeAnalysis.CSharp.Syntax.JoinClauseSyntax' does not exist in the implementation but it does exist in the contract.
+TypesMustExist : Type 'Microsoft.CodeAnalysis.CSharp.Syntax.JoinIntoClauseSyntax' does not exist in the implementation but it does exist in the contract.
+TypesMustExist : Type 'Microsoft.CodeAnalysis.CSharp.Syntax.LabeledStatementSyntax' does not exist in the implementation but it does exist in the contract.
+TypesMustExist : Type 'Microsoft.CodeAnalysis.CSharp.Syntax.LambdaExpressionSyntax' does not exist in the implementation but it does exist in the contract.
+TypesMustExist : Type '
```

Comment:
@weshaggard are we OK with all the ones below?

I spot checked most and they look ok.